cloudbridge/cloud/base/provider.py

```diff
@@ -94,6 +102,7 @@ def authenticate(self):
             self.security.key_pairs.list()
             return True
         except Exception as e:
+            log.debug("ProviderConnectionException ocurred")
```

Comment:
using log.exception here is recommended, as it will give a detailed stack trace.
```
log.exception("ProviderConnectionException occured" ,e)
```